lua/astrocommunity/test/nvim-coverage/init.lua

```diff
@@ -1,6 +1,33 @@
+local tests_prefix = "<Leader>T"
+local coverage_prefix = tests_prefix .. "C"
+
 return {
   "andythigpen/nvim-coverage",
   event = "User AstroFile",
-  dependencies = { "nvim-lua/plenary.nvim" },
-  opts = {},
+  opts = {
+    auto_reload = true,
+  },
+  dependencies = {
+    "nvim-lua/plenary.nvim",
+    {
+      "AstroNvim/astrolsp",
+      optional = true,
+      opts = {
+        mappings = {
+          n = {
+            -- INFO: Compatibility with `neotest` and `vim-test`
+            [tests_prefix] = { desc = "󰗇 Tests" },
+            [coverage_prefix] = { desc = " Coverage" },
+            [coverage_prefix .. "l"] = {
+              function() require("coverage").load(true) end,
+              desc = "Load and show coverage",
+            },
+            [coverage_prefix .. "t"] = { function() require("coverage").toggle() end, desc = "Toggle coverage" },
+            [coverage_prefix .. "s"] = { function() require("coverage").summary() end, desc = "Show coverage summary" },
+            [coverage_prefix .. "c"] = { function() require("coverage").clear() end, desc = "Clear coverage" },
+          },
+        },
+      },
+    },
```

Comment:
The key mapping should be using this style
https://github.com/AstroNvim/astrocommunity/blob/main/lua/astrocommunity/code-runner/overseer-nvim/init.lua#L38-L53

opts will be a function and you can move the local prefix to the function